package.json

```diff
@@ -177,6 +177,12 @@
       "hash": "Qmb1DA2A9LS2wR4FFweB4uEDomFsdmnw1VLawLE1yQzudj",
       "name": "base32",
       "version": "0.0.0"
+    },
+    {
+      "author": "kubuxu",
```

Comment:
I spent few hours tweaking this one around.
It has performance order of magnitude higher than other I've found.

Also original author was very smart about how he hashes and uses higher hashbits to expand lower bits for the multiple hash functions bloom filter requires.